src/app/api/v1/user/update-comm-option/route.ts

```diff
@@ -48,8 +48,13 @@ export async function POST(req: NextRequest) {
         default:
           allEmailsToPrimary = null;
       }
-      await setAllEmailsToPrimary(subscriber, allEmailsToPrimary);
-      await setMonthlyMonitorReport(subscriber, monthlyMonitorReport);
+
```

Comment:
I can't leave a comment on the relevant lines because they're unchanged, but it's best to update the type representing the request body to indicate that one of the properties should now be left empty:

```ts
export interface EmailUpdateCommOptionRequest {
  instantBreachAlerts?: EmailUpdateCommTypeOfOptions;
  monthlyMonitorReport?: boolean;
}
```

And then TS will show you what code to wrap in an `if (typeof <x> !== "undefined")`, and you won't need the custom type guard (`isEmailUpdateCommType`).